src/Qwiq.Core.Rest/QueryDefinition.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+
+namespace Microsoft.Qwiq.Client.Rest
+{
+    public class QueryDefinition : Qwiq.QueryDefinition
+    {
+        public QueryDefinition(QueryHierarchyItem queryDefinition)
```

Comment:
Should this not be `public`? The constructor exposes the underlying TFS type.

Reply:
Should be internal, will update.

Reply:
updated